portability-transfer/src/main/java/org/datatransferproject/transfer/JobProcessor.java

```diff
@@ -126,6 +126,9 @@ void processJob() {
       monitor.debug(
           () -> format("Finished copy for jobId: %s with %d error(s).", jobId, numErrors));
       success = errors.isEmpty();
+    } catch (DestinationMemoryFullException e) {
+      monitor.severe(() -> "Destination memory error processing jobId: " + jobId, e, EventCode.WORKER_JOB_ERRORED);
+      store.addFailureReasonToJob(jobId, "DESTINATION_FULL")
```

Comment:
Looks like the Travis CI failed because of:
```suggestion
      store.addFailureReasonToJob(jobId, "DESTINATION_FULL");
```